infoblox_client/objects.py

```diff
@@ -983,7 +983,8 @@ class EADefinition(InfobloxObject):
 
 
 class IPAddress(InfobloxObject):
-    _fields = ['network_view', 'ip_address', 'objects', 'network', 'status']
+    _fields = ['network_view', 'ip_address', 'objects', 'network', 'status',
+               'is_conflict', 'mac_address', 'types', 'usage', 'names']
```

Comment:
My only concern is `mac_address` - as far as I can tell, it doesn't apply to the underlying `ipv6address` object. I am not sure how best to address this...

@saiprasannasastry , @chinmayb  - please comment?

Reply:
@yuewko With my last update a proposal to fix the attributes difference